js/gamePlay.js

```diff
@@ -0,0 +1,74 @@
+$(function(){
+  initGame()
+  $(document).keydown(function(e){
+    var charPos = map[charY][charX]
+    counter=0
+    
+    //playerMovement
+  if(!isExit() && clicker){
+    switch(e.keyCode){
```

Comment:
What will be cons and pros of `switch` statement?

`if(!isExit() && clicker) {`
   `      if(e.keyCode === 37 && !checkXBlockLeft(charPos)) charMoveLeft()`
   `      if(e.keyCode === 38 && !checkYBlockTop(charPos)) charMoveUp()`
 `   if(e.keyCode === 39 && !checkXBlockRight(charPos)) charMoveRight()`
 `    if(e.keyCode === 40 && !checkYBlockBottom(charPos))charMoveDown()`
 `    mummyFollowChar()`
 `    meetMummy()`
`}`